roles/etcd/templates/etcd.j2

```diff
@@ -3,6 +3,9 @@
   --restart=on-failure:5 \
   --env-file=/etc/etcd.env \
   --net=host \
+  {% if ansible_architecture == "aarch64" -%}
```

Comment:
This is not a generic method.

Currently etcd only support `amd64` and `ppc64le` officially. For other platforms, such flag `ETCD_UNSUPPORTED_ARCH` is indispensable.

Shall we use `-e ETCD_UNSUPPORTED_ARCH={{ image_arch }}` here for other non amd64/ppc64le platforms?

Reply:
yes

Reply:
that said, we're missing all images for ppc64le, I would not aim for support it in kubespray.